spec/requests/create_a_booking_request_spec.rb

```diff
@@ -86,4 +93,32 @@ def and_the_service_responds_with_a_201
   def and_the_customer_receives_a_confirmation_email
     expect(ActionMailer::Base.deliveries.count).to eq(1)
   end
+
+  def when_an_invalid_booking_request_is_made
+    invalid_payload = {
+      'booking_request' => {
+        'location_id' => 'ac7112c3-e3cf-45cd-a8ff-9ba827b8e7ef',
+        'slots' => [
+          {
+            'date' => '2016-01-01',
+            'from' => '0900',
+            'to' => '1300',
+            'priority' => 1
+          }
+        ]
+      }
+    }
+
+    post api_v1_booking_requests_path(format: :json), invalid_payload.as_json
+  end
+
+  def then_the_service_responds_with_a_422
+    expect(response).to be_unprocessable
+  end
+
+  def and_the_errors_are_serialized_as_json
+    json = JSON.parse(response.body)
+
+    expect(json['errors']['name']).to be
```

Comment:
This verifies that `errors` are present, with a child `name`. Knowing how errors are structured, this gives me enough confidence to know things are being serialized / deserialized as I expect. `name` would not be present in the output unless it has validation errors, so I'm not sure what (other than personal preference, of course) adding a presence check buys us in this example?

I don't feel too strongly about this though, so if you do I'm happy to change.